images/prow-tests/Dockerfile

```diff
@@ -185,6 +185,9 @@ ARG JSONNET_VERSION=v0.18.0
 ARG COSIGN_VERSION=v1.13.0
 ARG TOOLBOX_VERSION=2051540f04c2a8fb5ad50c717345c2dd69a0b71e
 
+# Disable CGO libs to avoid error with too old version in base image
+ENV CGO_ENABLED=0
+
```

Comment:
@upodroid the image builds fine with `CGO_ENABLED`. Is there a way to verify Serving, Eventing other than merging the PR and runnings builds on the respective repos afterwards.